merchantapi/src/main/java/fi/hel/verkkokauppa/merchant/api/dto/MerchantDto.java

```diff
@@ -0,0 +1,24 @@
+package fi.hel.verkkokauppa.merchant.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import fi.hel.verkkokauppa.merchant.model.ConfigurationModel;
+import lombok.Data;
+import lombok.ToString;
+
+import javax.validation.constraints.NotBlank;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+@Data
+@ToString
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class MerchantDto {
+    // If given do not create new merchant
+    String merchantId;
```

Comment:
Should these fields be set as private for field access control? Lomboks @Data -annotation will create necessary getters and setters for the values.